src/com/palmergames/bukkit/util/ChatTools.java

```diff
@@ -89,10 +91,10 @@ public static String stripColour(String s) {
 
 	public static String formatTitle(String title) {
 
-		String line = ".oOo.__________________________________________________.oOo.";
+		String line = TownySettings.getLangString("title_banner");
 		int pivot = line.length() / 2;
-		String center = ".[ " + Colors.Yellow + title + Colors.Gold + " ].";
-		String out = Colors.Gold + line.substring(0, Math.max(0, (pivot - center.length() / 2)));
+		String center = String.format(TownySettings.getLangString("title_banner_center"), title);
+		String out = TownySettings.getLangString("title_banner_prefix") + line.substring(0, Math.max(0, (pivot - center.length() / 2)));
```

Comment:
Prefix and banner are separate so the cut is at the middle instead of 2 characters off due to the color code.